.env

```diff
@@ -33,6 +33,17 @@ CONFIG_IMAGE_NAME=bahmni/default-config
 CONFIG_IMAGE_TAG=latest
 CONFIG_VOLUME=bahmni-config
 
+# Metabase Environment variables
+METABASE_IMAGE_TAG=latest
+METABASE_DB_CONNECTION_URI=jdbc:postgresql://metabasedb:5432/metabase?user=metabase-user&password=password
```

Comment:
Also please check if you can use variables in the CONNECTION_URI as well so that credentials stays in one place.

Reply:
Instead of "latest" tag for metabase, can we fix on an actual released version of metabase? Latest is usually not always stable. Better to choose the latest released version and lock to that one for now. This is an external product, not one that we are developing.